assets/dev/js/frontend/handlers/background-slideshow.js

```diff
@@ -2,7 +2,7 @@ export default class BackgroundSlideshow extends elementorModules.frontend.handl
 	getDefaultSettings() {
 		return {
 			classes: {
-				swiperContainer: 'elementor-background-slideshow swiper-container',
+				swiperContainer: 'elementor-background-slideshow swiper',
```

Comment:
The class shouldn't be changed. It should be externalized to the elementorFrontend.config array and used from there, e.g.:
```suggestion
				swiperContainer: `elementor-background-slideshow ${ elementorFrontend.config.swiperClass }`,
```
or something similar

Reply:
@udidol Done